src/main/java/net/greatstart/dto/DtoUser.java

```diff
@@ -17,4 +18,7 @@
     private String password;
     @NotNull
     private String confirmPassword;
+    private String name;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "name" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)